src/frontend/packages/lib_video/src/components/live/Student/StudentLiveStarter/StudentLiveAdvertising/StudentLiveRegistration/RegistrationForm/index.tsx

```diff
@@ -191,101 +179,93 @@ export const RegistrationForm = ({
 
   return (
     <ThemeContext.Extend value={formTheme}>
-      <Form
-        value={values}
-        onChange={setValues}
-        onSubmit={async ({ value }) => {
-          if (!value.email) {
+      <form
+        onChange={(e) => {
+          const { name, value } = e.target as HTMLInputElement;
+          setValues((_value) => ({ ..._value, [name]: value }));
+        }}
+        onSubmit={(e) => {
+          e.preventDefault();
+
+          if (!values.email) {
             //  submit without email should not be possible since validation should fail
             return;
           }
 
-          let anonymousId: Maybe<string>;
-          if (!isValidToken) {
-            anonymousId = getAnonymousId();
-          }
-          let updatedLiveSession: LiveSession;
-          if (!liveSession) {
-            updatedLiveSession = await createLiveSession(
-              live.id,
-              value.email,
-              intl.locale,
-              anonymousId,
-            );
-          } else {
-            updatedLiveSession = await updateLiveSession(
-              liveSession,
-              intl.locale,
-              value.email,
-              true,
-              anonymousId,
-            );
-          }
+          const callWs = async () => {
```

Comment:
this is a nitpick, I think this name is not relevant. Why not simply calling it `submit` ?

Reply:
https://github.com/openfun/marsha/pull/2374/commits/8d917feeef6798f58ac5f51d02600191755712dc#diff-0f89eb57605035902a4f069eb0a8ca82424a9e91e63c22027048c2f809ba8029R195